codelab-elements/google-codelab-analytics/google_codelab_analytics_test.js

```diff
@@ -103,51 +101,47 @@ testSuite({
   },
 
   testCodelabGAIDAttr_InitsCodelabTracker() {
-    const analytics = new CodelabAnalytics();
-    analytics.setAttribute('gaid', 'UA-123');
     // Need to mock as we don't have window.ga.getAll()
-    const mockGetAll = mockControl.createFunctionMock('getAll');
-    const mockCreate = mockControl.createFunctionMock('create');
+    const mockGetAll = mockControl.createFunctionMock("getAll");
+    const mockCreate = mockControl.createFunctionMock("create");
     mockGetAll().$returns([]).$anyTimes();
     // Creates 2 trackers (because of codelab gaid attribute).
     mockCreate().$times(2);
 
-    window['ga'] = (...args) => {
-      if (['create', 'getAll'].indexOf(args[0]) !== -1) {
-        window['ga'][args[0]]();
+    window["ga"] = (...args) => {
+      if (["create", "getAll"].indexOf(args[0]) !== -1) {
+        window["ga"][args[0]]();
       }
     };
-    window['ga']['getAll'] = mockGetAll;
-    window['ga']['create'] = mockCreate;
+    window["ga"]["getAll"] = mockGetAll;
+    window["ga"]["create"] = mockCreate;
 
     mockControl.$replayAll();
 
     document.body.appendChild(analytics);
-    analytics.setAttribute('codelab-gaid', 'UA-456');
+    analytics.setAttribute("codelab-gaid", "UA-456");
```

Comment:
does this one need to be removed as well?

Reply:
Yep, we will have to remove all occurrences of `analytics.setAttribute` in the test file if we want them to be removed from prod

Reply:
Good catch @Mrc0113, thanks for finding this.